README.md

```diff
@@ -6,7 +6,8 @@ https://kyverno.io
 
 * This site makes use of the [Docsy](https://docsy.dev) theme.
   [Hugo Extended](https://gohugo.io/getting-started/installing#fetch-from-github) is required to render it.
-* `git clone https://github.com/kyverno/website kyverno-website/ --recurse-submodules`
+* Create a fork of the Kyverno website repository to your GitHub account.
+* `git clone https://github.com/{GITHUB-ID}/website kyverno-website/ --recurse-submodules`
```

Comment:
The `clone` would still have to reference `kyverno/website`:  `git clone https://github.com/kyverno/website kyverno-website/ --recurse-submodules`

Reply:
No, ideally you would want to clone your fork of `kyverno/website`. Not all maintainers have permissions to create additional branches in the `kyverno/website` repo. So the recommended process is a [fork & pull](https://movi.hashnode.dev/how-to-successfully-fork-clone-signoff-and-make-a-pull-request-ckdyt03sy06utjas18lx1cjer).

Reply:
Right, ok, so these are two separate instructions. Step one is create a fork (let's make it easy and link them to https://docs.github.com/en/get-started/quickstart/fork-a-repo). Step two is `git clone`. I'd mention that the `website` bit is by default but if a user changed this to a custom value then it should be `{REPO-NAME}`.

Reply:
Ok, the fork instruction is at the top of the page, but I can restate here for completeness.

Reply:
restructured this section, for clarity